src/components/task-teacher-review/index.cjsx

```diff
@@ -158,14 +179,37 @@ TaskTeacherReview = React.createClass
         </BS.Grid>
     </PinnedHeaderFooterCard>
 
-  onNextStep: ->
-    @goToStep(@state.currentStep + 1)()
-
 
 TaskTeacherReviewShell = React.createClass
   contextTypes:
     router: React.PropTypes.func
 
+  renderLoading: (refreshButton) ->
```

Comment:
Why not just use the default `renderLoading`? or use 2 nested `<Loadable>` items (one for the HW/iReading and one for all the problems)?